TheMET/TheMET/CatalogSectionCellData/CatalogSectionCellData.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CatalogSectionCellData.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 21/12/2023.
+//
+
+import Foundation
+import UIKit
+
+struct CatalogSectionCellData <T> {
+    
+    var sectionIdentificator: T
```

Comment:
this is a cell identifier, not a section one. Actually, it is better to just name it as `identifier`

Reply:
Done